src/Tizen.NUI/src/public/WindowEvent.cs

```diff
@@ -262,6 +266,31 @@ public partial class Window
             }
         }
 
+        /// <summary>
+        /// Keyboard Repeat Settings Changed
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<KeyboardRepeatSettingsChangedArgs> KeyboardRepeatSettingsChanged
```

Comment:
I think this KeyboardReatSettingsChanged event doesn't need to have argument because this KeyboardRepeatSettingsChangedArgs doesn't have any data. you may use just "EventHandler KeyboardRepeatSettingsChanged"

Reply:
Okay, just public event EventHandler KeyboardRepeatSettingsChanged
:)